build/testing/migration.go

```diff
@@ -0,0 +1,65 @@
+package testing
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"dagger.io/dagger"
+	"github.com/google/uuid"
+)
+
+func Migration(ctx context.Context, client *dagger.Client, base, flipt *dagger.Container) error {
+	dir := client.CacheVolume(fmt.Sprintf("flipt-state-%s", time.Now()))
+	latest, err := client.Container().
+		From("flipt/flipt:latest").
+		// support Flipt instances without /var/log/flipt configured
+		WithUser("root").
+		WithExec([]string{"mkdir", "-p", "/var/log/flipt"}).
+		WithExec([]string{"chown", "-R", "flipt:flipt", "/var/log/flipt"}).
+		WithUser("flipt").
+		WithEnvVariable("FLIPT_LOG_FILE", "/var/log/flipt/output.txt").
+		WithMountedCache("/var/opt/flipt", dir, dagger.ContainerWithMountedCacheOpts{
+			Owner: "flipt",
+		}).
+		WithExec([]string{"/flipt", "-v"}).
+		Sync(ctx)
+	if err != nil {
+		return err
+	}
+
+	// import testdata into latest Flipt instance
+	_, err = flipt.
+		WithFile("import.yaml", base.File("build/testing/integration/readonly/testdata/default.yaml")).
+		WithServiceBinding("flipt", latest.WithExec(nil)).
+		WithExec([]string{"/flipt", "import", "--address", "grpc://flipt:9000", "import.yaml"}).
+		Sync(ctx)
+	if err != nil {
+		return err
+	}
+
+	// run migration with edge Flipt build
+	flipt, err = flipt.
+		// WithEnvVariable("FLIPT_LOG_FILE", "/var/log/flipt/output.txt").
```

Comment:
Is this comment intentional?

Reply:
Ahh no, good spot, I was using this to reduce the log output volume at one point during debugging.